src/Examples/DigitalIdentity/DigitalIdentity/Controllers/DbsController.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Yoti.Auth;
+using Yoti.Auth.DigitalIdentity;
+using Yoti.Auth.DigitalIdentity.Policy;
+
+namespace DigitalIdentityExample.Controllers
+{
+    public class DbsController : Controller
+    {
+        private readonly string _clientSdkId;
+        private readonly ILogger _logger;
+        public DbsController(ILogger<DbsController> logger)
+        {
+            _logger = logger;
+
+            _clientSdkId = Environment.GetEnvironmentVariable("YOTI_CLIENT_SDK_ID");
+            _logger.LogInformation(string.Format("Yoti Client SDK ID='{0}'", _clientSdkId));
+        }
+       
+        // GET: /generate-share
+        [Route("dbs-share")]
+        public IActionResult DigitalIdentity()
+        {
+            try
+            {
+                string yotiKeyFilePath = Environment.GetEnvironmentVariable("YOTI_KEY_FILE_PATH");
+                _logger.LogInformation(
+                    string.Format(
+                        "yotiKeyFilePath='{0}'",
+                        yotiKeyFilePath));
+
+                StreamReader privateKeyStream = System.IO.File.OpenText(yotiKeyFilePath);
+
+                var yotiClient = new DigitalIdentityClient(_clientSdkId, privateKeyStream);
+
+                var givenNamesWantedAttribute = new WantedAttributeBuilder()
+                    .WithName("given_names")
+                    .WithOptional(false)
+                    .Build();
+                
+                var notification = new NotificationBuilder()
+                    .WithUrl("https://example.com/webhook")
+                    .WithMethod("POST")
+                    .WithVerifyTls(true)
+                    .Build();
+                
+                var policy = new PolicyBuilder()
+                    .WithIdentityProfileRequirements(new
+                    {
+                        trust_framework = "UK_TFIDA",
+                        scheme = new
+                        {
+                            type = "DBS",
+                            objective = "BASIC"
+                        }
+                    })
+                    .Build();
+
+                var sessionReq = new ShareSessionRequestBuilder().WithPolicy(policy)
```

Comment:
Sort of anti-pattern. Should move `WithPolicy` and `WithSubject` on new lines.

Reply:
solved